packages/terra-form/src/TextField.jsx

```diff
@@ -101,13 +107,15 @@ const TextField = ({
     error={error}
     help={help}
     isInline={isInline}
+    htmlFor={inputId}
     required={required}
     {...customProps}
   >
     <Input
       maxLength={maxLength}
       minLength={minLength}
       name={name}
+      id={inputId}
```

Comment:
Looks like I was wrong still, and mixed myself up with another issue that occurred. Try this commit out. https://github.com/cerner/terra-core/pull/511/commits/6b284e7359b926fbaac5bf1a52b05a495b1c5aad